ui/app/AppLayouts/Wallet/popups/simpleSend/SimpleSendModal.qml

```diff
@@ -208,13 +208,15 @@ StatusDialog {
         readonly property var selectedAssetEntry: ModelEntry {
             sourceModel: root.assetsModel
             key: "tokensKey"
+            cacheOnRemoval: true
             value: root.selectedTokenKey
             onItemChanged: d.setAssetInTokenSelector()
             onAvailableChanged: d.setAssetInTokenSelector()
         }
 
         // Holds if the asset entry is valid
-        readonly property bool selectedAssetEntryValid: selectedAssetEntry.available &&
+        readonly property bool selectedAssetEntryValid: (selectedAssetEntry.itemRemovedFromModel ||
```

Comment:
Can you please check scenario where a token is selected and then switch to a network on which the token is unavailable. The TokenSelector should show "Select asset" in this case. I think this use case will fail because of this change @Cuteivist